tasks/sign-binaries/sign-binaries.yaml

```diff
@@ -0,0 +1,150 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: sign-binaries
+  labels:
+    app.kubernetes.io/version: "0.2.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to sign mac and windows binaries using system specific signing hosts
+  params:
+    - name: quayURL
+      type: string
+      description: quay URL of the repo where content will be shared
+      default: quay.io/konflux-artifacts
+    - name: quaySecret
+      type: string
+      description: Secret to interact with Quay
+    - name: windowsCredentials
+      type: string
+      description: Secret to interact with the Windows signing host
+    - name: pipelineRunUid
+      type: string
+      description: Unique identifier for the pipeline run
+  volumes:
+    - name: windows-ssh-key-vol
+      secret:
+        secretName: windows-ssh-key
+  workspaces:
+    - name: data
+      description: Workspace to save the results to
+  results:
+    - name: unsignedWindowsDigest
+      type: string
+      description: |
+        The digest of the unsigned content pushed using ORAS for signing hosts
+    - name: signedWindowsDigest
+      type: string
+      description: |
+        The digest of the unsigned content pushed using ORAS for signing hosts
+  steps:
+    - name: push-unsigned-using-oras
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/bin/bash
```

Comment:
Nit: Maybe use `#!/usr/bin/env bash` to be consistent with other tasks? But if you prefer this, it's fine as well.

Also, maybe add `set -eux` or at least `set -x`? It usually doesn't hurt to have more details in the log.

Reply:
What we started doing in other places is start with `-x` and then for the snippet where some secret would be leaked, you add `set +x` above it and then `set -x` below. That way you still have more logging most of the time, but avoid leaking anything. WDYT of that?